thorlcr/master/thmastermain.cpp

```diff
@@ -378,13 +378,26 @@ bool checkClusterRelicateDAFS(IGroup *grp)
     PROGLOG("Checking cluster replicate nodes");
     SocketEndpointArray epa;
     grp->getSocketEndpoints(epa);
-    ForEachItemIn(i1,epa) {
+    SocketEndpointArray epau;
+    ForEachItemIn(i1,epa)
+    {
         epa.element(i1).port = getDaliServixPort();
+        bool add = true;
+        ForEachItemIn(i2,epau)
+        {
+            if (epa.element(i1).equals(epa.element(i2)))
+            {
+                add = false;
+                break;
+            }
+        }
+        if (add)
+            epau.append(epa.element(i1));
```

Comment:
I will move this to validateNodes.  There are also some other places where an asyncFor is used to call getRemoteVersion and we also do not dedup the host:port.  I am not sure of the context of all those and if its correct to always dedup.